linebot/webhook/parse.go

```diff
@@ -15,14 +15,30 @@ var (
 	ErrInvalidSignature = errors.New("invalid signature")
 )
 
-// ParseRequest func
-func ParseRequest(channelSecret string, r *http.Request) (*CallbackRequest, error) {
+type ParseOption struct {
+	// SkipSignatureValidation is a function that determines whether to skip
+	// webhook signature verification.
+	//
+	// If the function returns true, the signature verification step is skipped.
+	// This can be useful in scenarios such as when you're in the process of updating
+	// the channel secret and need to temporarily bypass verification to avoid disruptions.
+	SkipSignatureValidation func() bool
+}
```

Comment:
When adding another option later, would it always require a major version upgrade? I think it's better to ensure the code works seamlessly with users' existing builds, even when adding another option.

Reply:
To make sure this change doesn't break existing code that uses the current line-bot-sdk-go, I also added tests for the existing `ParseRequest` (without options).